### PR TITLE
WIP: 43 design review

### PR DESCRIPTION
**Pull request by helenb**

### CODE REVIEW

**Comment by helenb** [comment]
@chris-lawton please could you review the changes so far?

File: src/components/author-block/author-block.module.scss [hunk 1]
```diff
@@ -9,9 +9,14 @@
     }
 
     &Image {
-        width: 80px !important;
-        height: 80px !important;
+        width: 55px;
```
Comment (chris-lawton) [inline src/components/author-block/author-block.module.scss]: happy to see the number of `!important`'s being reduced!

**Comment by chris-lawton** [comment]
Pull down and looking good. I'm finding that the scroll behaviour when click on the in-page nav links not very smooth e.g. home page > scroll down > click between service/client/process. I don't think it's related to any of the work done here but may be worth checking on your end to see if it's just a problem i'm seeing.

**Comment by helenb** [comment]
@chris-lawton re the scrolling do you find the same on https://tbx-staging.netlify.com/? I have found that on my local build any animations are quite janky, but once deployed to staging they are super fast.

File: src/components/author-block/author-block.module.scss (hunk 1 above)
Comment (helenb) [reply src/components/author-block/author-block.module.scss]: Yes, trying to tidy up as I go along :)

Comment (helenb) [reply src/components/author-block/author-block.module.scss]: (I don't think any of them are necessary)

**Comment by chris-lawton** [comment]
Ah yes, that's a massive improvement on staging! Must be some config with the local build?

**Pull request merged**